Teams.Tests/ManageTeamsServiceTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Moq;
+using Teams.Security;
+using NUnit.Framework;
+using Teams.Data;
+using Teams.Models;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+using Teams.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace Teams.Tests
+{
+    [TestFixture]
+    class ManageTeamsServiceTest
+    {
+        public IManageTeamsService _mangeTeamsService;
+        public ICurrentUser _currentUser;
+        public Mock<IRepository<Team, int>> _teamRepository;
+        public Mock<IRepository<TeamMember, int>> _teamMemberRepository;
+        private Mock<IHttpContextAccessor> _httpContextAccessor;
```

Comment:
please use empty lines between methods and properties, methods and methods, properties and fields
will be more readable